content/learn/book/welcome/why-bevy/_index.md

```diff
@@ -5,8 +5,71 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: discuss the cool features and design philosophy
+There are plently of great reasons to use Bevy, but here are just a few:
+
+### ECS 
+**TL;DR: Better workflow and abstraction in your code.**
+
+**LR:**
+
+Bevy has been designed to use a flexible data system in order to provide a smart, usable workflow. The engine uses an **Entity Component System** (ECS) to acheive this goal. In traditional game engines, a singular object (entity) contains its logic and components in one system. This is known as an object-oriented entity system.
+
+This has its benifits, however, in general, it is being phased out by ECSs. Instead of containing all the systems and components in a singular entity, in ECS, all of this is abstracted into structs (in Rust) and functions. Entities are defined as unique integers. 
```

Comment:
> it is being phased out by ECSs

I *personally* think this is likely to be true, but is probably a bit inflammatory :p

> Entities are defined as unique integers

This is mostly an implementation detail: you could use strings or chars or PNG files as entities just fine if you *really* wanted. The critical bit is that they're unique identifiers that you can use to organize and quickly look up the data attached to a particular "unit", whether that's a tangible game object or something more abstract.